### ISSUE TITLE
Regression: Reserved words in concise methods throwing error

### ISSUE THREAD

**Issue by thejameskyle** [6to5/6to5#401]
```js
var foo = {
  try() {},
  throw() {},
  catch() {},
  switch() {},
  // etc.
}
```
```js
SyntaxError: Unexpected token catch
    at Function (native)
    at compile (https://6to5.org/repl.html:200:33)
```

This was working in 1.x.

**Comment by sebmck** [comment]
Fixed in `master` after the `traceur` branch merge.

**Comment by sebmck** [comment]
Thanks! Fixed as of 2.6.0
